live-objects/src/main/kotlin/io/ably/lib/objects/DefaultLiveObjects.kt

```diff
@@ -94,6 +89,21 @@ internal class DefaultLiveObjects(private val channelName: String, internal val
     TODO("Not yet implemented")
   }
 
+  override fun on(event: ObjectsStateEvent, listener: ObjectsStateChange.Listener): ObjectsSubscription =
+    objectsManager.on(event, listener)
+
+  override fun off(listener: ObjectsStateChange.Listener) = objectsManager.off(listener)
+
+  override fun offAll() = objectsManager.offAll()
```

Comment:
let's not expose `offAll()` at least for now. `offAll`, `unsubscribeAll` - are "gigantic foot guns" (c) Andy

Reply:
Same exists in case of ably-js https://github.com/ably/ably-js/blob/837afe6585f68bd9fabf6bf8fd14b723258e9b30/src/plugins/objects/objects.ts#L187, need to disusss with team yes

Reply:
Also, we have `unsubscribeAll` in case of `object-subscriptions`.  You can check https://github.com/ably/specification/pull/346